airbyte-integrations/connectors/source-pexels-api/source_pexels_api/pexels_api.yaml

```diff
@@ -0,0 +1,97 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://api.pexels.com"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "Authorization"
+      api_token: "{{ config['api_key'] }}"
+
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+
+  base_stream:
+    schema_loader:
+      type: JsonSchema
+      file_path: "./source_pexels_api/schemas/{{ options['name'] }}.json"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+  photos_search_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "photos_search"
+      path: "/v1/search?query={{ config['query'] or 'people' }}&orientation={{ config['orientation'] or 'landscape'}}&size={{ config['size'] or 'large'}}&color={{ config['color'] or 'orange'}}&locale={{ config['locale'] or 'en-US'}}&page={{ config['page'] or '1'}}&per_page={{ config['per_page'] or '15'}}"
```

Comment:
I don't think this s correct. Check how to add request parameters with other connectors implementations or readint the docs: https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/request-options

Reply:
Done